client/my-sites/themes/install-theme-button.scss

```diff
@@ -1,3 +1,3 @@
 .has-no-sidebar .themes__upload-button {
-    margin-right: 32px;
-}
\ No newline at end of file
+	margin-right: 32px;
```

Comment:
Hmm, I feel like the linter should have caught this. Looks like it's reverted from using tab indentation to spaces.

Reply:
I don't remember the linter running here, but it seems to have corrected to tab indentation? At least, I can't seem to revert it.

<img width="1359" alt="Screenshot 2021-07-15 at 21 27 41" src="https://user-images.githubusercontent.com/43215253/125853375-ba3dc187-fa38-4271-bb38-21dbd8363c68.png">

Reply:
🤷‍♀️ Obvs not a blocker. Ignoring. :D